Client.js

```diff
@@ -38,25 +38,17 @@ function get(resourceId, qs, cb) {
     cb = qs
     qs = {}
   }
-  // We do not use url.resolve becuase if the resourceId contains a leading slash,
-  // we would end up with an incorrect url.
-  var requestUrl = this.resourcePath + '/' + resourceId
-  return this.client.request({method: 'GET', uri: encodeUri(requestUrl), qs: qs}, cb)
+  
+  return this.client.request({method: 'GET', uri: encodeUri(this.resourcePath, resourceId), qs: qs}, cb)
```

Comment:
this.resourcePath is locally defined in this file (lines 20 + 25) so we only really need to de/encode the resource id